backend/src/Service/Order/OrderService.php

```diff
@@ -65,10 +77,21 @@ public function getStoreOrders($userId): ?array
      * @param $id
      * @return OrdersResponse
      */
-    public function getSpecificOrderForStore($id): ?OrdersResponse
+    public function getSpecificOrderForStore(int $id): ?OrdersResponse
     {
         $order = $this->orderManager->getSpecificOrderForStore($id);
 
+        $order['images'] = $this->getImageParams($order['images'], $this->params.$order['images'], $this->params);
```

Comment:
After using UploadFileHelperService, we will not need this statement for getting image params.